apps/webClient/src/components/View/index.tsx

```diff
@@ -41,48 +112,64 @@ const useStyles = makeStyles(theme => ({
     flexGrow: 1,
   },
   redPageWrapper: {
-    backgroundColor: RED,
-    background: RED_GRADIENT,
+    backgroundColor: DARK_ORANGE,
+    backgroundImage: RED_GRADIENT,
   },
   greenPageWrapper: {
-    backgroundColor: GREEN,
-    background: GREEN_GRADIENT,
+    backgroundColor: DARK_GREEN,
+    backgroundImage: GREEN_GRADIENT,
   },
   bluePageWrapper: {
-    backgroundColor: BLUE,
-    background: BLUE_GRADIENT,
+    backgroundColor: DARK_BLUE,
+    backgroundImage: BLUE_GRADIENT,
   },
   yellowPageWrapper: {
-    backgroundColor: YELLOW,
-    background: YELLOW_GRADIENT,
+    backgroundColor: DARK_YELLOW,
+    backgroundImage: YELLOW_GRADIENT,
+  },
+  "@keyframes homeToOurWork": {
```

Comment:
The map data structure will be useful because we will be able to generate the keyframes automatically:

```
[`@keyframes ${someValue}`]: {
  ...
}
```